revise readme to focus on the user (#80)
conda-incubator/conda-project: README.md

[![codecov](https://codecov.io/gh/conda-incubator/conda-project/branch/main/graph/badge.svg?token=XNRS8JKT75)](https://codecov.io/gh/conda-incubator/conda-project)
[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/conda-incubator/conda-project/main.svg)](https://results.pre-commit.ci/latest/github/conda-incubator/conda-project/main)

Tool for encapsulating, running, and reproducing projects with conda environments.

This package is intended as a successor to [Anaconda Project](https://github.com/Anaconda-Platform/anaconda-project).

## Why?

> Sharing your work is more than sharing the script file or notebook with your code. To help your fellow users it is

Reviewer: ```suggestion
Sharing your work is more than sharing your code in a script file or notebook. To make your work properly reproducible, it is necessary to include the list of required third-party dependencies, specifications for how to run your code, and
any other files that it may need.
```

> necessary to include the list of required third-party dependencies, specifications for how to run your code, and

Reviewer: ```suggestion
```

> any other files that it may need.

Reviewer: ```suggestion
```

See [8 Levels of Reproduciblity](https://www.anaconda.com/blog/8-levels-of-reproducibility) for an in-depth

> discussion of the differences between "It works fo me." to "I've made sure that anyone can reliably execute my work."

Reviewer: ```suggestion
discussion of the differences between "It works for me." to "I've made sure that anyone can reliably execute my work."
```

> Conda Project provides a framework that guides you to ensuring a high degree of reproducibility in the projects you

Reviewer: ```suggestion
Conda Project is a framework that aims to help you to ensure a high degree of reproducibility in the projects you
```

create.

## Installation

> You can install conda-project using the conda package manager

Reviewer: ```suggestion
You can install conda-project using the conda package manager:
```

```

> conda install -c defusco conda-project

Author: Yes, once we get 0.1.0 out I'll update to conda-forge

```

## Quick start

> Let's start a new project using Python, Pandas, and Jupyter Notebooks. First, create the project directory,
> install the conda environment, and launch Jupyter. The commands below will work on terminals in Mac, Linux, and Window.

Reviewer: Just figured out how to properly use this 🤦 

```suggestion
Let's start a new project using Python, Pandas, and Jupyter Notebooks. 
The commands below will work on terminals in Mac, Linux, and Windows.
```

For Windows you can use either `cmd.exe` or Powershell.

> ```

Reviewer: ```suggestion
We first create a directory and initialize a new project, which will create a new conda environment and lock the dependencies:

```
```

```

The goal of Conda Project is to maintain a conda enviroment specifically for the new `my-project` directory.
You'll see that this directory contains it's own `environment.yml` file a [Conda Lock](https://conda.github.io/conda-lock/) file and a `conda-project.yml`
file. You can learn more about these files in the [User Guide](https://conda-incubator.github.io/conda-project/user_guide.html)

```
(base) > tree ./
├── conda-project.yml
├── default.conda-lock.yml
├── environment.yml
```

You can activate the environment, which will install packages locally to this project according to the lock file.
Notice that after running `conda project activate` the shell prompt switches to `(default)`, which is the name
of the local environment for this project.

```
(base) > cd my-project
(base) > conda project activate

Downloading and Extracting Packages

Downloading and Extracting Packages

Preparing transaction: done
Verifying transaction: done
Executing transaction: done
environment created at /Users/adefusco/Development/conda-incubator/conda-project/examples/my-project/envs/default
## Project environment default activated in a new shell.
## Exit this shell to de-activate.
```

> And in the activated environment you can launch editors or run commands.

Reviewer: ```suggestion
And in the activated environment you can launch editors or run commands:
```
